platform/windows/display_server_windows.cpp

```diff
@@ -1877,11 +1877,33 @@ LRESULT DisplayServerWindows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARA
 			break;
 		}
 		case WM_ACTIVATE: { // Watch For Window Activate Message
-			saved_wparam = wParam;
-			saved_lparam = lParam;
 
```

Comment:
This adds an empty line at the start of the code block.

Reply:
Fixed